src/themes/OLH/templates/core/homepage_elements/search_bar.html

```diff
@@ -12,7 +12,10 @@
             <input id="sort" name="sort" type="hidden" value="relevance">
             <input id="title" name="title" type="hidden" value="on">
             <input id="abstract" name="abstract" type="hidden" value="on">
-              <span class="input-group-label"><span class="fa fa-search"></span></span>
+              <span class="input-group-label">
+                <i aria-hidden="true" class="fa fa-search"></i>
+                <span class="sr-only">{% trans 'Search' %}</span>
+              </span>
               <input name="article_search" class="input-group-field" type="text">
```

Comment:
Similar to with the OLH theme template for search, this needs a more robust label explicitly tied to the input. In this case I think the `<span class="sr-only">` should be changed to a `label` and tied to the input with a "for" attribute.